scanners/android/android.go

```diff
@@ -129,6 +137,15 @@ func parseProjects(searchDir string, projectRoots []string) ([]Project, error) {
 	return projects, nil
 }
 
+func usesKotlinBuildScripts(projectRoot string) bool {
```

Comment:
Is this going to work when the Gradle project is in a subfolder? For example, React Native projects put the whole Gradle project in `./android`, or a monorepo where the Android client is in a subfolder.

Reply:
I have checked and it will work. The Android scanner uses this method to traverse the directory layout from a starting folder:
https://github.com/bitrise-io/bitrise-init/blob/c9266f2c255a700a64a8e9a6352785fe747e169c/scanners/android/utility.go#L88

And it will find all of possible Android projects. From here it will filter down further by only using those potential project roots where there is a gradlew wrapper:
https://github.com/bitrise-io/bitrise-init/blob/8ea7233cb5f2763d4f7bcaebdc4be240acd5b23e/scanners/android/android.go#L104

And based on my research there is always a `build.gradle` or `build.gradle.kts` file (and the settings equivalent) next to the wrapper. Is that a correct assumption?

Reply:
Yes, that's correct, I was thinking about the pattern when the entire Gradle project is in a subfolder. But I'm glad it's going to work!